src/ami-set.js

```diff
@@ -41,4 +41,29 @@ let AmiSet = base.Entity.configure({
   },
 });
 
+/**
+ * Load all the knwon Amis
+ */
+AmiSet.listAmiSets = async function () {
+
+  let amislist = [];
```

Comment:
Normally, we'd camel case this into `amisList`, but given that a list is an aggregation(in this case, a list), I'd normally use a singular for the first part, so `amiList`, but since we're talking about ami sets and not amis, I'd call it `amiSetList`.